prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml

```diff
@@ -0,0 +1,412 @@
+job_template: &job_template
+  branches:
+  - "^release-1.2$"
+  decorate: true
+  path_alias: istio.io/istio
+
+istio_container: &istio_container
+  image: gcr.io/istio-testing/istio-builder:v20181008-db31a9fd
+  # Docker in Docker
+  securityContext:
+    privileged: true
+  resources:
+    requests:
+      memory: "512Mi"
+      cpu: "500m"
+    limits:
+      memory: "24Gi"
+      cpu: "7000m"
+
+presubmits:
+
+  istio/istio:
+
+  - name: istio-unit-tests
+    <<: *job_template
+    context: prow/istio-unit-tests.sh
+    always_run: true
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/istio-unit-tests.sh
+      nodeSelector:
+        testing: build-pool
+
+  - name: istio-integ-local-tests
+    <<: *job_template
+    context: prow/istio-integ-local-tests.sh
+    always_run: true
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/istio-integ-local-tests.sh
+      nodeSelector:
+        testing: build-pool
+
+  - name: istio-presubmit
+    <<: *job_template
+    context: prow/istio-presubmit.sh
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - <<: *istio_container
+        command:
+        - entrypoint
+        - prow/istio-presubmit.sh
+      nodeSelector:
+        testing: build-pool
+    run_after_success:
+    - name: istio-integ-k8s-tests
+      <<: *job_template
+      optional: true
+      context: prow/istio-integ-k8s-tests.sh
```

Comment:
Sorry I meant this one , presubmit. We split to things like prow/integ-galley-k8s-presubmit-tests.sh

Reply:
updated.